song-server/src/main/java/bio/overture/song/server/controller/UploadController.java

```diff
@@ -56,20 +57,7 @@ public SubmitResponse submit(
       @RequestHeader(value = AUTHORIZATION, required = false) final String accessToken,
       @PathVariable("studyId") String studyId,
       @RequestBody @Valid String json_payload) {
-    return uploadService.submit(studyId, json_payload, false);
+    return uploadService.submit(studyId, json_payload);
```

Comment:
Should check to make sure the client code and sdk examples match the server changes.

Specifically, I noticed song-python-sdk/examples/example_upload.py, line 76  still has "ignore_analysis_id_collisions" as a parameter to api.save.

Reply:
thank you for the reminder. I completely forgot lol